api/client/service.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "google/rpc/status.proto";
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message StartRequest {
+  CommandLineOptions options = 1;
+}
+
+// TODO(oschaaf): Not implemented yet.
+message UpdateRequest {
+}
+
+// TODO(oschaaf): Not implemented yet.
+message CancellationRequest {
+}
+
+message ExecutionRequest {
+  oneof command_specific_options {
+    StartRequest start_request = 1;
```

Comment:
You may want to set `option (validate.required) = true;` on this.

Reply:
Done